gui/src/components/Onboarding/Wizard.tsx

```diff
@@ -51,10 +52,12 @@ export function OnboardingWizard({ closeOnboarding }: Props) {
 
   return (
     <Box px={3}>
-      <Stack direction="row" py={1.5} spacing={1} alignItems="center">
-        <Logo width={40} />
-        <Typography>Iron Wallet</Typography>
-      </Stack>
+      <Box data-tauri-drag-region="true" pt={4}></Box>
+      {type !== "Darwin" && (
+        <Box>
+          <Logo width={40} />
+        </Box>
+      )}
```

Comment:
why not hide it globally? this feels like it's not a macOS specific thing

Reply:
I don't know 🤷 in macos we already have the icon on the menubar, and we don't usually see it on the app itself. I'm not sure about how it works in other systems

Reply:
I was checking slack, telegram, obsidian and discord. The only one that shows the logo is Discord, but because it's the same button that goes to DMs

should be fine to remove I think

Reply:
👍 done